### DISCUSSION TITLE
Not clickable page http://<....>:8080/home

### DISCUSSION THREAD

**Discussion by beckss1** [apache/airflow#29994] [Q&A]
Hi.,
I as Oracle DBA asked install Airflow on RedHat 7.9 (64 bit). Done all steps for installation and creating user admin. 
After launching web server from entering with admin password  I redirected to http://<....>:8080/home page and all links (for example Dataset, Security and all radio buttons are not clickable)
Please advise what is a problem or probably what Ii need to check.
if need I will upload screen with my problem.
Thanks and best regards,
Stas

**Comment by potiuk** [comment]
You have not explained how you installed it. There are various ways. And yes uploading what you see might be adviseable. What you could see (if for example you installed from sources) you might have no assets compiled (see INSTALL in sources) or your user is not admin or gas not enough permissions (see users section in our docs)

**Reply by beckss1** [reply]
Hi.,
Thanks for answer.
1. installation was done according to documentation of Airflow. -> 
. # Airflow needs a home. `~/airflow` is the default, but you can put it


# somewhere else if you prefer (optional)
export AIRFLOW_HOME=~/airflow

# Install Airflow using the constraints file
AIRFLOW_VERSION=2.5.1
PYTHON_VERSION="$(python3.7 --version | cut -d " " -f 2 | cut -d "." -f 1-2)"
# For example: 3.7
CONSTRAINT_URL="https://raw.githubusercontent.com/apache/airflow/constraints-${AIRFLOW_VERSION}/constraints-${PYTHON_VERSION}.txt"
# For example: https://raw.githubusercontent.com/apache/airflow/constraints-2.5.1/constraints-3.7.txt
pip install "apache-airflow==${AIRFLOW_VERSION}" --constraint "${CONSTRAINT_URL}"

2. Admin also installed -> 
 airflow users create --username admin --firstname <......>--lastname <....>. --role Admin --email <.....>
(<....> please as filled )


Please explain what you mean -> 
What you could see (if for example you installed from sources) you might have no assets compiled (see INSTALL in sources) 

Thanks and regards,
Stas

**Comment by beckss1** [comment]
<img width="927" alt="Airflow1" src="https://user-images.githubusercontent.com/8437982/224025211-57285f9f-b969-4914-a999-27e664014d57.PNG">
<img width="945" alt="Airflow2" src="https://user-images.githubusercontent.com/8437982/224025280-8db134f3-7b5a-49ec-83a7-f13968bc0fb8.PNG">

**Reply by potiuk** [reply]
Then everything is fine. Likey your browser has some problem if you cannot click the links and if your re sure your user has admin rights. Try different browser

**Reply by beckss1** [reply]
Thanks for update.
You correct - after moving to Mozilla browser all links are clickable.
Continue reading manual Airflow.
Best regards,
Stas
